types/open-location-code/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "module": "node16",
+        "lib": ["es6", "dom"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
This is not guaranteed to be in a tsconfig, so it needs to be removed to make sure projects without it can use these types.

Reply:
It says in the readme not to use esModuleInterop, I missed that.
```
esModuleInterop and allowSyntheticDefaultImports are not allowed in your tsconfig.json.
```